source/getting-started/managing-dependencies.md

```diff
@@ -0,0 +1,129 @@
+As you're developing your Ember app, you'll likely run into common scenarios that aren't addressed by Ember itself, such as authentication or using SASS for your stylesheets. Ember CLI provides a common format called [Ember Addons](#toc_addons) for distributing reusable libraries to solve these problems. Additionally, you may want to make use of front-end dependencies like a CSS framework on a JavaScript datepicker that aren't specific to Ember apps. Ember CLI supports installing these packages through the standard [Bower package manager](#toc_bower).
+
+## Addons
+
+Ember Addons are installed using NPM (eg. `npm install --save-dev ember-cli-sass`). Addons
+may bring in other dependencies by modifying your project's `bower.json` file
+automatically.
+
+You can find listings of addons on [Ember Observer](http://emberobserver.com).
+
+## Bower
+
+Ember CLI uses the [Bower](http://bower.io) package manager, making it easy
+to keep your front-end dependencies up to date. The Bower configuration file, `bower.json`, is located at the root of your Ember
+CLI project, and lists the dependencies for your project. Executing `bower install`
+will install all of the dependencies listed in `bower.json` in one step.
+
+Ember CLI watches `bower.json` for changes. Thus it reloads your app if you
+install new dependencies via `bower install <dependencies> --save`.
+
+## Other assets
+
+Assets not available as an addon or Bower package should be placed in the `vendor` folder in your project.
+
+## Compiling Assets
+
+When you're using Bower dependencies that are not included in an addon, you will
+have to instruct Ember CLI to include your assets in the build. This is done using
+the asset manifest file `ember-cli-build.js`. You should only try to import assets
+located in the `bower_components` and `vendor` folders.
+
+### Javascript assets exporting a globals
+
+Provide the asset path as the first and only argument.
+
+```ember-cli-build.js
+app.import('bower_components/moment/moment.js');
+```
+
+The globals provided by the asset ( `moment` in this example can now
```

Comment:
There's an extra space after the first `(`. Also, the closing `)` is missing after `example`.